runconfig/parse.go

```diff
@@ -65,6 +65,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 		flCpuShares       = cmd.Int64([]string{"c", "-cpu-shares"}, 0, "CPU shares (relative weight)")
 		flCpusetCpus      = cmd.String([]string{"#-cpuset", "-cpuset-cpus"}, "", "CPUs in which to allow execution (0-3, 0,1)")
 		flCpusetMems      = cmd.String([]string{"-cpuset-mems"}, "", "MEMs in which to allow execution (0-3, 0,1)")
+		flBlkioWeight     = cmd.Int64([]string{"-blkio-weight"}, 0, "Block IO relative weight, range from 10 to 1000")
```

Comment:
Yes, that's a problem, `docker inspect` will show 0, right now a lot of configs have this problem. So my understanding is:
`docker inspect` shows container/image configs, not states, so they are likely different.